packages/create-sitecore-jss/src/templates/nextjs/src/Layout.tsx

```diff
@@ -38,16 +37,22 @@ const Layout = ({ sitecoreContext: { route } }: LayoutProps): JSX.Element => {
       <Navigation />
       {/* root placeholder for the app, which we add components to using route data */}
       <div className="container">
-        {route && <Placeholder name="<%- appPrefix ? `${appName}-` : '' %>jss-main" rendering={route} />}
+        {layoutData.sitecore.route && (
+          <Placeholder
+            name="<%- appPrefix ? `${appName}-` : '' %>jss-main"
+            rendering={layoutData.sitecore.route}
+          />
+        )}
       </div>
     </>
   );
 };
 
 const propsAreEqual = (prevProps: LayoutProps, nextProps: LayoutProps) => {
-  if (deepEqual(prevProps.sitecoreContext.route, nextProps.sitecoreContext.route)) return true;
+  if (deepEqual(prevProps.layoutData.sitecore.route, nextProps.layoutData.sitecore.route))
+    return true;
 
   return false;
 };
 
-export default withSitecoreContext()(React.memo(Layout, propsAreEqual));
+export default React.memo(Layout, propsAreEqual);
```

Comment:
Since we are binding directly to `LayoutData`, we shouldn't need to use memo+deepEqual to prevent double rendering, right? I thought I remember us testing this and it was fine.

Reply:
Done